src/util/KvAuth0.js

```diff
@@ -90,7 +90,7 @@ export default class KvAuth0 {
 	}
 
 	// Open a popup window to the login page
-	popupLogin() {
+	popupLogin(prompt) {
```

Comment:
How about doing this as a options object that is spread after any defaults we want to set? like
```
webAuth.popup.authorize({
	prompt: 'none'
	popupOptions: { .. },
	...options
}
```